pkg/plugins/jenkins/main.go

```diff
@@ -54,51 +51,19 @@ func (j *Jenkins) Validate() (err error) {
 
 // GetVersions fetch every jenkins version from the maven repository
 func GetVersions() (latest string, versions []string, err error) {
-	m := maven.Maven{
-		URL:        "repo.jenkins-ci.org",
-		Repository: "releases",
-		GroupID:    "org.jenkins-ci.main",
-		ArtifactID: "jenkins-war",
-	}
-
-	URL := fmt.Sprintf("https://%s/%s/%s/%s/maven-metadata.xml",
-		m.URL,
-		m.Repository,
-		strings.ReplaceAll(m.GroupID, ".", "/"),
-		m.ArtifactID)
-
-	req, err := http.NewRequest("GET", URL, nil)
+	m := mavenmetadata.New("https://repo.jenkins-ci.org/releases/org/jenkins-ci/main/jenkins-war/maven-metadata.xml")
```

Comment:
I think it would be better to move this information to a package constant.

Reply:
Good idea. Pointed in https://github.com/updatecli/updatecli/pull/482 (as it is centered on jenkins plugin)